backend/controllers/order.controller.js

```diff
@@ -10,7 +11,9 @@ import constants from "../config/constants.js";
 const createOrder = asyncHandler(async (req, res) => {
   try {
     const orderItems = req.body;
-    const itemPrice = orderItems
+    const user = await User.findById(req.user._id);
```

Comment:
I've changed this so only auth user can create orders. @giangnguyen13  @hanguy2806